app/Helpers/helpers.php

```diff
@@ -0,0 +1,16 @@
+<?php
+function bannerImg()
+{
+    $context = stream_context_create(['http' => ['header' => 'Connection: close\r\n']]);
+    $url = 'https://viblo.asia/';
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Helpers/helpers.php</br>**Line**: 7</br>Variable "match_leng" is not in valid camel caps format